src/ontology/slurp/ordo.tsv

```diff
@@ -1,12 +1,12 @@
-MONDO:0970961	fibroneural non-saccular limited dorsal myeloschisis	Orphanet:645310	MONDO:equivalentTo	Fibroneural non-saccular limited dorsal myeloschisis	A rare non-saccular limited dorsal myeloschisis, in which the stalk histology is characterized by skin tissue with dermo-epidermal epithelium in continuity with a subcutaneous fibrocollagenous tract containing various ectopic tissues.	MONDO:8000031|MONDO:0968989
-MONDO:0970962	terminal myelocystocele	Orphanet:645337	MONDO:equivalentTo	Terminal myelocystocele	A rare closed spinal dysraphism characterized by a myelocystocele at the termination of the spinal cord. It may be an isolated anomaly or be associated with other defects, including sacral agenesis, anorectal and genitourinary anomalies. The conus is not identifiable. The myelocystocele sac may have a significant lipomatous component (terminal lipomyelocystocele).	MONDO:0968988|MONDO:8000034|MONDO:8000030|MONDO:0017077
-MONDO:0970963	saccular limited dorsal myeloschisis	Orphanet:645354	MONDO:equivalentTo	Saccular limited dorsal myeloschisis	A rare form of limited dorsal myeloschisis (LDM), characterized by the stalk attached to the apex of a fully epithelialized meningocele. Chiari II malformation is not present.	MONDO:0968988|MONDO:8000034|MONDO:8000030
-MONDO:0970964	myelic limited dorsal malformation	Orphanet:645378	MONDO:equivalentTo	Myelic limited dorsal malformation	A rare intermediate form of open dysraphism between myelomeningocele and saccular limited dorsal myeloschisis without fulfilling the characteristics of one of these two diagnosis, characterized by stretched neurulated spinal cord attached at the dome of a sac. Partial cerebral signs of open dysraphism can be observed and the meningocele is usually poorly epithelialized.	MONDO:0968988|MONDO:8000034|MONDO:8000030
+mondo_id	mondo_label	xref	xref_source	original_label	definition	parents	subset
+ID	LABEL	A oboInOwl:hasDbXref	>A oboInOwl:source SPLIT=|		A IAO:0000115	SC %	
+MONDO:0968990	genetic central precocious puberty in male	Orphanet:650097	MONDO:equivalentTo	Genetic central precocious puberty in male		MONDO:8000031|MONDO:0958356	subtype of a disorder
+MONDO:0968991	non-genetic central precocious puberty in male	Orphanet:650102	MONDO:equivalentTo	Non-genetic central precocious puberty in male		MONDO:0958356|MONDO:8000031	subtype of a disorder
```

Comment:
@joeflack4 - double check how you determine if something is a subtype, disorder or something else - remember yesterdays discussion about the ordo subsets and the problem with the "component".

Reply:
I checked it, looks good. Using the correct file!